submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const numOfFriends=20;
+const apiUrl = `https://randomuser.me/api/?results=${numOfFriends}`;
+const friends = document.querySelector(".friends");
+let friendsData;
+let arrForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function makeContainerEmpty() {
+        let innerText = " "; 
+        friends.innerHTML = innerText.replace(innerText, " ");    
+}
+
+function fetchFriend() {
+        fetch(apiUrl)
+                .then(response => response.json())
```

Comment:
https://www.tjvantoll.com/2015/09/13/fetch-and-errors/

Reply:
I don't get it. Should I just catch error with some text about it f.e.
`.catch(err => {
                        alert("Ops...Something went wrong");`
?

Reply:
No, you should handle response status because `catch` will not catch "all cases"

Reply:
I think I've got it.